### ISSUE TITLE
代码中为什么没有测试阶段

### ISSUE THREAD

**Issue by icoderzqliu** [betterAndTogether/IncreProtoNet#5]
请问为什么代码中直接拿测试数据集来做验证？验证集似乎没有使用到。请问可以解释一下一下处理分割好的数据集分别都是用在哪个阶段嘛？谢谢！

**Comment by betterAndTogether** [comment]
您具体可以仔细看看论文数据集的划分以及模型介绍：
Three parts: training set which consists of 54 relations (i.e., base relations R_base) each with 550 instances; validation set which consists of 54 relations (i.e., base relations R_base) each with 50 instances and 10 relations (i.e., novel relations in validation stage) each with 700 instances; and testing set which consists of 54 relations (i.e., base relations R_base) each with 100 instances and 16 relations (i.e., novel relations Rnovel in testing stage) each with 700 instances. There are no-overlapping instances between training, validation and testing dataset.

**Comment by icoderzqliu** [comment]
> 您具体可以仔细看看论文数据集的划分以及模型介绍： Three parts: training set which consists of 54 relations (i.e., base relations R_base) each with 550 instances; validation set which consists of 54 relations (i.e., base relations R_base) each with 50 instances and 10 relations (i.e., novel relations in validation stage) each with 700 instances; and testing set which consists of 54 relations (i.e., base relations R_base) each with 100 instances and 16 relations (i.e., novel relations Rnovel in testing stage) each with 700 instances. There are no-overlapping instances between training, validation and testing dataset.

感谢您的回复~我能理解论文数据集的划分，但是在仓库代码中并没有看到test阶段，而且base_val_fewrel.json，novel_val_fewrel.json这两个文件并没有使用到。请问论文中的结果是不是验证阶段的结果？麻烦您的解惑，谢谢！

**Comment by betterAndTogether** [comment]
> > 您具体可以仔细看看论文数据集的划分以及模型介绍： Three parts: training set which consists of 54 relations (i.e., base relations R_base) each with 550 instances; validation set which consists of 54 relations (i.e., base relations R_base) each with 50 instances and 10 relations (i.e., novel relations in validation stage) each with 700 instances; and testing set which consists of 54 relations (i.e., base relations R_base) each with 100 instances and 16 relations (i.e., novel relations Rnovel in testing stage) each with 700 instances. There are no-overlapping instances between training, validation and testing dataset.
> 
> 感谢您的回复~我能理解论文数据集的划分，但是在仓库代码中并没有看到test阶段，而且base_val_fewrel.json，novel_val_fewrel.json这两个文件并没有使用到。请问论文中的结果是不是验证阶段的结果？麻烦您的解惑，谢谢！

该版本代码，您可以修改对应参数：test_baseData_path以及test_novelData_path, 即可进行验证;

**Comment by icoderzqliu** [comment]
> > > 您具体可以仔细看看论文数据集的划分以及模型介绍： Three parts: training set which consists of 54 relations (i.e., base relations R_base) each with 550 instances; validation set which consists of 54 relations (i.e., base relations R_base) each with 50 instances and 10 relations (i.e., novel relations in validation stage) each with 700 instances; and testing set which consists of 54 relations (i.e., base relations R_base) each with 100 instances and 16 relations (i.e., novel relations Rnovel in testing stage) each with 700 instances. There are no-overlapping instances between training, validation and testing dataset.
> > 
> > 
> > 感谢您的回复~我能理解论文数据集的划分，但是在仓库代码中并没有看到test阶段，而且base_val_fewrel.json，novel_val_fewrel.json这两个文件并没有使用到。请问论文中的结果是不是验证阶段的结果？麻烦您的解惑，谢谢！
> 
> 该版本代码，您可以修改对应参数：test_baseData_path以及test_novelData_path, 即可进行验证;

感谢回复~
